sqrl-execute/sqrl-execute-http/sqrl-execute-http-vertx/src/main/java/com/datasqrl/graphql/io/SinkProducer.java

```diff
@@ -0,0 +1,8 @@
+package com.datasqrl.graphql.io;
+
+import io.vertx.core.Future;
+import java.util.Map;
+
+public interface SinkProducer {
+  public Future<SinkResult> send(Map<String, Object> entry);
```

Comment:
It dependent on vertx classes when it was in http-core. I moved it to vertx to remove that dependency. I think it is tricky to refactor the vertx dependency out because Vertx uses it's own Future class.
There might be a way to map that but for now we should leave it tied to vertx.